src/foam/nanos/dig/DIG.js

```diff
@@ -17,6 +17,7 @@ foam.CLASS({
 
   implements: [
     'foam.mlang.Expressions',
+    'foam.nanos.auth.CreatedByAware'
```

Comment:
Didn't realize we didn't have createdBy, created, lastModified, ... 
Please add the other three.